mkosi/config.py

```diff
@@ -1851,7 +1887,10 @@ def parse(self, argv: Optional[Sequence[str]] = None) -> tuple[MkosiArgs, tuple[
         # infrastructure scripts rather than image-specific configuration.
         self.backward_compat_stubs(namespace)
 
-        return args, tuple(load_config(ns) for ns in presets)
+        presets = list(load_config(ns) for ns in presets)
```

Comment:
```suggestion
        presets = [load_config(ns) for ns in presets]
```
Just more readily identifiable as a list.